src/markdown-pages/docs/cli-command-reference/cli-command-reference.md

```diff
@@ -1,11 +1,11 @@
 ---
 path: "/docs/cli-command-reference/cli-command-reference"
-title: "Using the Galasa CLI"
+title: "Getting started using the Galasa CLI"
 ---
 
 Use the Galasa command line interface (Galasa CLI) to help you complete tasks, for example, submitting and monitoring Galasa test runs. You can use the same set of Galasa CLI commands to run a given task, regardless of the technology that you are using to run your pipeline. 
 
-Galasa CLI commands start with `galasactl`. You can view the full list of Galasa CLI commands in the [Galasa cli repository](https://github.com/galasa-dev/cli/tree/main/docs/generated).
+Galasa CLI commands start with `galasactl` on Unix or Mac. You can view the full list of Galasa CLI commands in the [Galasa cli repository](https://github.com/galasa-dev/cli/tree/main/docs/generated).
```

Comment:
Are we mentioning `galasactl.exe` on Windows here also ? Or maybe this piece should be moved to how to install the CLI

Reply:
I added `galasactl.exe`